services/QuillLMS/app/controllers/concerns/diagnostic_reports.rb

```diff
@@ -40,13 +40,13 @@ def summarize_correct_skills(number_correct, number_incorrect)
   end
 
   # rubocop:disable Metrics/CyclomaticComplexity
-  private def set_activity_sessions_and_assigned_students_for_activity_classroom_and_unit(activity_id, classroom_id, unit_id=nil, hashify_activity_sessions: false)
+  def set_activity_sessions_and_assigned_students_for_activity_classroom_and_unit(activity_id, classroom_id, unit_id=nil, hashify_activity_sessions: false)
```

Comment:
Is there a reason we're changing the access modifier here other than for testing?

Reply:
Just testing. I could have used `Object.send` instead, but I felt that this method was complex and used enough for it to be a first class citizen in the module.

Reply:
True.  I'm noticing the method seemed to ignore private before your change anyway:  https://github.com/empirical-org/Empirical-Core/pull/8919/files#diff-52f3cb88c5a79c7f77d63a8165222bdabd3eee0f61ccd1859ba6bd8f9aa2e423R65